src/applications/representatives/containers/Dashboard.jsx

```diff
@@ -2,6 +2,28 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'react-router';
 
+import { Notifications } from '../components/Notifications/Notifications';
+
+// import { RequiredLoginView } from '@department-of-veterans-affairs/platform-user/RequiredLoginView';
+
+const dummyNotificationData = [
+  {
+    name: 'John Smith',
+    id: 12345,
+    date: '24 JAN 2024 09:00AM',
```

Comment:
Sounds good! I was thinking that if we formatted the [found this after a quick search](https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years) it in the dummy data it would be correctly formatted in the table which is always our goal. Then when we are actually getting data from the backend we already have that standard of what that data should look like that we can then add a helper to get it back to that.